internal/api/v1/user/totp/totp_test.go

```diff
@@ -0,0 +1,146 @@
+package totp_test
+
+import (
+	"encoding/hex"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	bip32 "github.com/bitcoin-sv/go-sdk/compat/bip32"
+	client "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/totp"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/xpriv"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/clienttest"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateTotpForContact(t *testing.T) {
```

Comment:
Following the naming convention used in other tests, please rename it to: _TestClient_GenerateTotpForContact_. Also please apply this change in another places.